help/configure-authentication-methods.md

```diff
@@ -2,27 +2,56 @@
 
 {!owner-only.md!}
 
-By default, Zulip allows logging in via email/password as well as
-various social authentication providers like Google, GitHub, GitLab,
-and Apple. You can restrict users to logging in via only a subset of
-these methods.
+You can choose which authentication methods to enable for users to log in to
+your organization. The following options are available on all
+[plans](https://zulip.com/plans/):
 
-LDAP and various custom SSO login methods are currently restricted to
-self-hosted Zulip organizations only. SAML authentication is supported
-by Zulip Cloud but requires contacting support@zulip.com to configure it.
+- Email and password
+- Social authentication: Google, GitHub, GitLab, Apple
 
-**Note:** If you are running your own server,
-[read this](https://zulip.readthedocs.io/en/stable/production/authentication-methods.html)
-first. Server configuration is needed for several of the authentication
-methods listed above.
+The following options are available for organizations on Zulip Cloud Standard,
+Zulip Cloud Plus, and all self-hosted Zulip servers:
+
+- Oauth2 with Azure Active Directory
+
+The following options are available for organizations on Zulip Cloud Plus, and all self-hosted Zulip servers:
+
+- [SAML authentication](/help/saml-authentication), including Okta, OneLogin, AzureAD, Keycloak, Auth0
+- [SCIM provisioning](/help/scim)
+
+The following authentication and identity management options are available for
+all self-hosted servers. If you are interested in one of these options for a
+Zulip Cloud organization, contact [support@zulip.com](mailto:support@zulip.com)
+to inquire.
+
+- [AD/LDAP user
+  sync](https://zulip.readthedocs.io/en/stable/production/authentication-methods.html#ldap-including-active-directory)
+- [AD/LDAP group
+  sync](https://zulip.readthedocs.io/en/stable/production/authentication-methods.html#ldap-including-active-directory)
+- [OpenID
+  Connect](https://zulip.readthedocs.io/en/stable/production/authentication-methods.html#openid-connect)
+- [Custom authentication
+  options](https://python-social-auth.readthedocs.io/en/latest/backends/index.html#social-backends)
+  with python-social-auth
 
 ### Configure authentication methods
 
+!!! warn ""
+
+    For self-hosted organizations, some authentication options require
+    that you first [configure your
+    server](https://zulip.readthedocs.io/en/stable/production/authentication-methods.html)
+    to support the option.
+
 {start_tabs}
 
 {settings_tab|auth-methods}
 
-2. Toggle the checkboxes next to the available login options.
+1. To use SAML authentication or SCIM provisioning, Zulip Cloud organizations
+   must upgrade to [Zulip Cloud Plus](https://zulip.com/plans/), and contact
+   [support@zulip.com](mailto:support@zulip.com) to enable these methods.
```

Comment:
Is this the right phrasing, given that currently you actually can't upgrade to Plus on your own and https://zulip.com/plans/  tells you to just contact sales? So maybe something like

> To use SAML ..... organization must contact [support@zulip.com](mailto:support@zulip.com) to purchase a [Zulip Cloud Plus](https://zulip.com/plans/) plan and configure/enable (not sure which verb is better) these features <probably better word than "methods", since SCIM is not an auth method?>.

Reply:
I think its all right, since we plan to make it possible to upgrade to Plus on your own, and changing this page would be an extra thing to remember?

Reply:
@timabbott leaving it up to you.

Reply:
Yeah I think it's probably most future-proof as written.